atlas/portal/pkg/svc/zserver.go

```diff
@@ -0,0 +1,219 @@
+package svc
+
+import (
+	models "atlas/portal/internal"
+	"atlas/portal/pkg/pb"
+	responderpb "atlas/responder/pkg/pb"
+	"context"
+	"errors"
+	"github.com/golang/protobuf/ptypes/empty"
+	"github.com/sirupsen/logrus"
+	"google.golang.org/grpc"
+	"time"
+)
+
+const (
+	// version is the current version of the service
+	version            = "0.0.1"
+	invalidServiceName = "invalid service name"
+	emptyRequest       = "empty Request"
+	success            = "success"
+	errorMsg           = "error"
+	errorMissingResp   = "missing repository response error"
+	hiddenUptimeMsg    = "uptime is hidden, mode = false"
+	portal             = "portal"
+	responder          = "responder"
+	storage            = "storage"
+)
+
+type Portal struct {
+	pb.PortalServer
+	client responderpb.ResponderClient
+	logger *logrus.Logger
+	portal models.Service
+}
+
+func getPortal() models.Service {
+	return models.Service{
+		ServiceName:          "portal",
+		ServiceDesc:          "portal service desc",
+		ServiceUptime:        time.Now(),
```

Comment:
```suggestion
		ServiceUptime:        time.Now().UTC(),
```
Please use time.Now() only for internal purpose.
for example: if you communicate with services, they can be in difference time zone.

Reply:
Done